src/components/GameItem.js

```diff
@@ -4,22 +4,41 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false,
+    };
+  }
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  onGameItemClick = () => {
+    console.log('Item clicked!');
+    if (!this.state.clicked && this.props.item === 'litter') this.props.incrementScoreFunc();
+    this.setState({
+      clicked: true,
+    })
+  }
+
   render() {
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.item];
 
+    let status = '';
+    if (this.state.clicked) {
+      status = (this.props.item === 'litter') ? 'spotted-litter' : 'spotted-nature';
+    }
```

Comment:
Extremely minor nitpick: Since this code is included in `render`, it could be run many times with the same result. A slightly cleaner solution might be to put this logic in the click handler, store the class name in `state`, and read that value here instead of recomputing it.